ProjectLighthouse/PlayerData/Profiles/User.cs

```diff
@@ -136,6 +136,9 @@ private int PhotosWithMe()
     [JsonIgnore]
     public string PlanetHashLBPVita { get; set; } = "";
 
+    [JsonIgnore]
+    public string PlanetHashCrossControl { get; set; } = "";
```

Comment:
This name sounds extra long, maybe it could be reduced to something like `PlanetHashCC`. It kind of removes the implied usage though so it might be better to keep this.